### ISSUE TITLE
Back Test Engine Fails To Execute Strategy

### ISSUE THREAD

**Issue by benjamin-keil-crypto-worx** [benjamin-keil-crypto-worx/botfox#4]
Steps To Reproduce

visit: https://benjamin-keil-crypto-worx.github.io/botfox-wiki/#/pages/quickStart?id=starting-a-simple-bot

Enter
````js
let {BitFoxEngine, SuperTrend} = require("bitfox").bitfox;

let options = {
    "requiredCandles": 200,
    "sidePreference": "long",
    "backtest": true,
    "pollRate":10,
    "public": true,
    "exchange": "bybit",
    "symbol": "ADAUSDT",
    "timeframe": "15m",
    "amount": 100,
    "profitPct": 0.003,
    "fee": 0.02,
    "key": "FAKE_KEY",
    "secret": "FAKE_SECRET",
    "life": false,
    "interval": 10
}

(async  () =>{
    
    // Initialize the Engine via options instance 
    let engine = BitFoxEngine.init(options);

    // Alternatively initialize engine via the builder interface (Recommended instantiation flow!!)
    let builder = require("bitfox").bitfox.builder; // import the builder interface
    let engine2 = builder()
        .requiredCandles(200)
        .sidePreference("long")
        .backtest(true)
        .pollRate(10)
        .public(true)
        .exchange("bybit")
        .symbol("ADAUSDT")
        .timeframe("15m")
        .amount(100)
        .profitPct(0.003)
        .fee(0.02)
        .key("FAKE_KEY")
        .secret("FAKE_SECRET")
        .life(false)
        .interval(10)
        .build(); 
        
    // Set up the Exchage Client
    await engine.setupAndLoadClient()
    
    // Leverage A Strategy from botfoxes strategy repository
    engine.applyStrategy(SuperTrend)
    // or Alternatively 
    let bitfox = require("bitfox").bitfox;
    engine.applyStrategy(bitfox.SuperTrend)
    
    // Set Up Event Handlers 
    engine.on('onMessage', (eventArgs) => {
        console.log(eventArgs)
    });
    engine.on('onError', (eventArgs) => {
        console.log(eventArgs)
    });
    engine.on('onOrderPlaced', (eventArgs) => {
        console.log(eventArgs)
    });
    engine.on('onOrderFilled', (eventArgs) => {
        console.log(eventArgs)
    });
    engine.on('onTradeComplete', (eventArgs) => {
        console.log(eventArgs)
    });
    engine.on('onStopLossTriggered', (eventArgs) => {
        console.log(eventArgs)
    });
    // SetUp Custom Event Handler (You need to fire the event yourself BotFox doesn't know about your Custom Event")
    engine.on('MyCustomEvent', (eventArgs) => {
        console.log(eventArgs)
    });
    
    // You could get the Event handler and fire an event by using below code
    engine.getEventEmitter().fireEvent("MyCustomEvent", {customEventData:"YourCustomEventData"})
    
    // Start The BotFox Engine
    await engine.run();
})();
````
Expected Behavior:

It should run without error and Print out Back testing result

Actual Behavior

````shell
C:\Users\benja\develop\bf-testing\node_modules\bitfox\strategies\SuperTrend.js:38
            return this.getStrategyResult(this.state,{value:data[isBackTest ? _index : data.length - 1].value}, Su);
                                                                                                                ^

ReferenceError: Su is not defined
    at SuperTrend.run (C:\Users\benja\develop\bf-testing\node_modules\bitfox\strategies\SuperTrend.js:38:113)
    at BackTest.backTest (C:\Users\benja\develop\bf-testing\node_modules\bitfox\engine\BackTest.js:39:46)
    at async BitFox.runBackTest (C:\Users\benja\develop\bf-testing\node_modules\bitfox\engine\BitFox.js:416:9)
    at async BitFox.run (C:\Users\benja\develop\bf-testing\node_modules\bitfox\engine\BitFox.js:338:13)
    at async C:\Users\benja\develop\bf-testing\bitfox-tests-simple-bot.js:82:5

Node.js v18.12.1
````

**Comment by benjamin-keil-crypto-worx** [comment]
Need to ensure that we remove some leftover code Snippet in SuperTrend Strategy

![image](https://user-images.githubusercontent.com/123937063/230438580-9a7c351c-ded1-449c-8da3-512d31e22ea6.png)

The ``Su`` should not be in there!

**Comment by benjamin-keil-crypto-worx** [comment]
Applied fix in #6
